### ISSUE TITLE
Feature request: Add "Privacy Mode"

### ISSUE THREAD

**Issue by danjm** [MetaMask/metamask-extension#10653]
This would be a setting defaulted to off.

With the setting on, we could blur out numbers and transaction totals with CSS (https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur());

This would enable people to keep account data (like balances and transaction amounts) private, even when others are seeing their screen.

**Comment by darkwing** [comment]
Design requirements:

* We need an easily viewable button, preferably on the home screen, to quickly toggle on and off privacy mode
* Direction about the degree to which to blur or style the tokens and their fiat currency value

Engineering requirements:

* Wire up the toggling of the button, ensuring it hides token and fiat values on both the home screen and the individual token screen.
* Investigate where else we may need to apply the privacy blur.  Swaps?  Send?

**Comment by danjm** [comment]
Do you think the toggle button could just be in settings? (For MVP, to minimize design requirements and not use up any precious home screen real estate)

**Comment by danjm** [comment]
Is the "blur" just a bit of css?

**Comment by darkwing** [comment]
> Do you think the toggle button could just be in settings? (For MVP, to minimize design requirements and not use up any precious home screen real estate)

Yeah, we could probably do that.  Long term it really needs to be accessible almost always though.

> Is the "blur" just a bit of css?

Yep, as well as maybe setting the color to grey.  Alternatively we could use a custom font if we want more visual control.
